plugins/explore-backend/src/index.ts

```diff
@@ -34,6 +34,7 @@ import type {
   ToolDocumentCollatorFactoryOptions as _ToolDocumentCollatorFactoryOptions,
 } from '@backstage/plugin-search-backend-module-explore';
 
+export { explorePlugin } from './plugin';
```

Comment:
please make this a default export, [like this](https://github.com/backstage/backstage/blob/7bc38d3ef6b14500a37c01556ecfe3173991a7d4/plugins/devtools-backend/src/index.ts#L25).

Reply:
done